Gnosis: fixed rules that formerly needed `split()`, to work without it

denis-bogdanas opened the pull request:
Now `split()` is not used by any project.

daejunpark commented:
what did you change? the only makefile? i don't see why the makefile change fixes the split issue. what am i missing here?

denis-bogdanas commented:
Yes only the makefile. Former `checkSignatures-loop-success` is now divided into checkSignatures-loop-success-middle and checkSignatures-loop-success-end, to replace the effect of `split()`. But checkSignatures-loop-success-middle needs the full checkSignatures-loop-success to pass. So I added checkSignatures-loop-success-trusted to the list of specs it needs. That's all. Most changes were done in the previous PR. I introduced checkSignatures-loop-success-middle and checkSignatures-loop-success-end there, deleted split(), but commented these new specs because they were failing or not yet tested.
